lib/Server.js

```diff
@@ -25,7 +25,7 @@ const createLog = require('./createLog');
 const OptionsValidationError = require('./OptionsValidationError');
 const optionsSchema = require('./optionsSchema.json');
 
-const clientStats = { errorDetails: false };
+const clientStats = { errorDetails: false, chunkSortMode: 'none', chunks: false };
```

Comment:
I think you can even go with

``` js
{ all: false, assets: true, warnings: true, errors: true, errorDetails: false }
```

since these are the only fields used.

Reply:
will do!

Reply:
Updated